Code/Tools/ProjectManager/Source/DownloadController.cpp

```diff
@@ -83,26 +90,27 @@ namespace O3DE::ProjectManager
             {
                 QMessageBox gemDownloadError;
                 gemDownloadError.setIcon(QMessageBox::Critical);
-                gemDownloadError.setWindowTitle(tr("Gem download"));
+                gemDownloadError.setWindowTitle(tr("Object download"));
                 gemDownloadError.setText(result);
                 gemDownloadError.setDetailedText(detailedError);
                 gemDownloadError.exec();
```

Comment:
consider renaming the QMessage box `downloadError` and the window title to say `Download failed`